src/coreclr/vm/loaderallocator.cpp

```diff
@@ -1118,7 +1118,8 @@ void LoaderAllocator::Init(BaseDomain *pDomain, BYTE *pExecutableHeapMemory)
 
 #if !defined(HOST_64BIT)
     // Make sure that we reserve as little as possible on 32-bit to save address space
-    _ASSERTE(dwTotalReserveMemSize <= VIRTUAL_ALLOC_RESERVE_GRANULARITY);
+    // We cannot reserve less than needed
+    //_ASSERTE(dwTotalReserveMemSize <= VIRTUAL_ALLOC_RESERVE_GRANULARITY);
```

Comment:
Delete this instead of uncommenting it?

Reply:
This is an accidental leftover, thank you for noticing.